relation/core.md

```diff
@@ -22,3 +22,38 @@ The standard error is 3 standard deviation of the mean of the sample.
 ### relation:StandardDeviation {#StandardDeviation}
  
  Dispersion of the values of a random variable around its expected value.
+
+### relation:Confidence {#MarginOfError}
+ 
+Use `Confidence` to specify uncertainty in the estimate. It can be used multiple times,
+in which case add `dcterms:type` to each dimension relation to specify the type of confidence 
+(Margin of Error, Confidence Interval, etc.).
+ 
+#### relation:ConfidenceUpperBound and relation:ConfidenceLowerBound {#ConfidenceBounds}
+
+In case of asymmetric confidence intervals, use `relation:ConfidenceUpperBound` together with 
+`relation:ConfidenceLowerBound` to specify the upper and lower bounds of the confidence interval.
+
+```turtle
+[
+  sh:property [ 
+    sh:path ex:measurement-dimension ;
+  ] ;
+  sh:property [
+    sh:path ex:lower-confidence-dimension ;  
+    meta:dimensionRelation [ 
+      a relation:ConfidenceLowerBound ;
+      dcterms:type "Confidence interval" ;
```

Comment:
I don't remember if this was already discussed and agreed-upon elsewhere, but the recommended values for `dcterms:type` may be URIs from a controlled vocabulary

Reply:
You're right. It is stated by the [vocabulary itself](https://prefix.zazuko.com/dcterms:type)

> Recommended practice is to use a controlled vocabulary such as the DCMI Type Vocabulary [[DCMI-TYPE](http://dublincore.org/documents/dcmi-type-vocabulary/)]